src/util/screening-selector.js

```diff
@@ -6,27 +6,51 @@ export async function getScreeningInstrument(client, patientId) {
   if (!client) throw new Error("invalid FHIR client provided");
   let screeningInstrument = "";
   if (patientId) {
-    const carePlan = await client.request(`CarePlan?subject=Patient/${patientId}&_sort=-_lastUpdated`);
+    const carePlan = await client.request(
+      `CarePlan?subject=Patient/${patientId}&_sort=-_lastUpdated`
+    );
     if (carePlan && carePlan.entry && carePlan.entry.length) {
       const resources = carePlan.entry;
       // TODO: need to figure out which one is the next questionnaire to do
       // For now, assumming the first one in the activity array is the next questionnaire to do?
       if (resources.length) {
-        const activities = resources[0].resource? resources[0].resource.activity: null;
-        if (activities) {
-          const canonicalIdentifier = activities[0].detail && activities[0].detail.instantiatesCanonical ? activities[0].detail.instantiatesCanonical[0]: null;
-          if (canonicalIdentifier) {
-            //assuming instantiatesCanonical references questionnaire in format like this:  Questionnaire/[Questionnaire ID]
-            screeningInstrument = canonicalIdentifier.split("/")[1];
+        let activities = [];
+        // gather activities from careplan(s)
+        resources.forEach((item) => {
+          if (item.resource.activity) {
+            activities = [...activities, ...item.resource.activity];
+          }
+        });
+        // loop through activities that contains instantiatesCanonical
+        if (activities.length) {
```

Comment:
@achen2401  It looks like the change in this PR is to support multiple activities within the care plan, not multiple care plans per se? Just trying to reconcile this w/ the title and description of this PR.

Reply:
thanks, @mcjustin.  I updated the description and title of the PR to better describe the issue.   When the application searches for the patient's care plan,  the server can potentially return multiple entries as described in the description of this PR.  So it effectively has to look through them all.

Reply:
Thanks for clarifying - this makes sense, and as I read your change more closely, I see that the code does iterate over each of the CarePlan(s) for which this patient is a subject, and adds each of the CarePlan(s)' activities to a queue. 

Also, I hadn't considered that we might have more than one CarePlan for each patient, but I can see that being the case if we have more than one applicable PlanDefinition in the future.